tests/js/assets/js/yii.validation.test.js

```diff
@@ -0,0 +1,1598 @@
+describe('yii.validation', function () {
+    var VALIDATOR_SUCCESS_MESSAGE = 'should leave messages as is';
+    var VALIDATOR_ERROR_MESSAGE = 'should extend messages with according error(s)';
```

Comment:
@nkovacs. Sorry, I thought you are suggesting to use one generic message instead of two. Yes, I can modify message to be more precise.